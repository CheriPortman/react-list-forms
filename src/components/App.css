.App {
  display: grid;
  grid-template-columns: 1fr 3fr;
  grid-column-gap: 2rem;
  font-size: 5rem;
  background-color: purple;
  color:  rgb(153, 105, 16);
}
