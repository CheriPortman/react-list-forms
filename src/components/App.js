import React from 'react';
import Photos from './photos/Photos';

import threeLoves from '../assets/three-loves.jpg';
import stitch from '../assets/stitch-on-blue.jpg';

export default function App() {
  const photos = [
    threeLoves,
    stitch
  ];

  return (
    <>
      <Photos photos={photos}/>
    </>
  );
}
