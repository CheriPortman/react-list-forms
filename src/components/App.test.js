import React from 'react';
import App from './App';
import { shallow } from 'enzyme';

describe('App Componenet', () => {
  it('renders the app', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
