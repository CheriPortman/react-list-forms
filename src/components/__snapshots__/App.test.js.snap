// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Componenet renders the app 1`] = `
<Fragment>
  <ErrorBoundry>
    <Bad />
  </ErrorBoundry>
  <Good />
</Fragment>
`;
